docker/openboxes.yml

```diff
@@ -0,0 +1,6 @@
+dataSource.dbCreate: none
+dataSource.url: jdbc:mysql://${DB_HOST:host.docker.internal}:${DB_PORT:3306}/${DB_NAME:openboxes}?useSSL=false
```

Comment:
@jmiranda would you say we should keep it as is with multiple variables or just have a single DATASOURCE_URL env variable to set the whole url?

Reply:
@jmiranda .env file is only used for the runtime environment. The openboxes.yml is a static part of the build environment, it is contained in the image, and it just references the env variables.

Docker doesn't require the openboxes.yml - we can just use env variables sourced from .env. After thinking, l reckon I could get rid of openboxes.yml, just document instructions to set up .env (DATASOURCE_URL primarily) properly. And docs should mention that these 3 (DB) env variables have to be provided to the runtime environment.

I didn't consider AWS Elastic Beanstalk or ECS support at this point. I think it would be great to support it though since we are at the point of having a working OB Docker image. Supporting Elastic Beanstalk involves a few other changes (not a lot) to be done to the Docker-related files to make it work - I think this can be done in a separate ticket.